nimbleclient/v1/restclient.py

```diff
@@ -392,3 +433,38 @@ def perform_bulk_resource_action(self, resource, action, **params):
 
         resp = self.post(f"{self._ENDPOINTS[resource]}/actions/{action}", **params)
         return resp['data'] if 'data' in resp else resp
+
+    def job_handler(self, job_id, job_timeout, retry_interval=5):
+
+        # Use global/default job timeout if unspecified in the request
+        timeout_sec = job_timeout if job_timeout is not None else self.job_timeout
+
+        now = datetime.datetime.now()
+        expiry_time = now + datetime.timedelta(0, timeout_sec)
+        logging.debug(f"job_id: [{job_id}], job_timeout: [{timeout_sec} seconds], current_time: [{now}], job_retry_expiry_time: [{expiry_time}], retry_interval: [{retry_interval} seconds]")
+        while 1:
+            try:
+                # Get job status from Array
+                response = self.get_resource('jobs', job_id)
+                logging.debug("JOB STATUS RESPONSE: %s", json.dumps(response, indent=4))
+                if (response['state'] == '1done'):
```

Comment:
Double check that this conditional is not a typo. Is it `1done` not `done`?

Reply:
oh right.. I will change that to 'done'. Thanks!